install_config/master_node_configuration.adoc

```diff
@@ -960,15 +961,22 @@ related files in the specified directory:
 
 [options="nowrap"]
 ----
-$ oadm create-node-config --node-dir=/openshift.local.config/node-<node_hostname> --node=<node_hostname> --hostnames=<hostname>,<ip_address>
+$ oadm create-node-config \ 
+--node-dir=/openshift.local.config/node-<node_hostname> \
+--node=<node_hostname> \
+--hostnames=<node_hostname>,<ip_address> \
+--certificate-authority="/path/to/ca.crt" \
+--signer-cert="/path/to/ca.crt" \
+--signer-key="/path/to/ca.key" 
+--signer-serial="/path/to/ca.serial.txt" \ 
+--node-client-certificate-authority="/path/to/ca.crt" 
 ----
 
-For the `--hostnames` option in the above command, use a comma-delimited list of
-every host name or IP address you want server certificates to be valid for. The
-above command also assumes that certificate files are located in an
-*_openshift.local.config/master/_* directory. If they are not, you can include
-options to specify their location. Run the command with the `-h` option to see
-details.
+When creating node configuration files, the `--hostnames` option accepts a
+comma-delimited list of every host name or IP address you want server
+certificates to be valid for. The above command also assumes that certificate
+files are located in an *_openshift.local.config/master/_* directory. If they
```

Comment:
not sure this line is needed any more, since we explicitly specify the path to the certs above